### PR TITLE
feature: finish task3

### PR DESCRIPTION
**Pull request by adele-ty**

### CODE REVIEW

**Comment by adele-ty** [comment]
/chatgpt clean my code in this PR

File: src/main.ts [hunk 1]
```diff
@@ -1,22 +1,40 @@
 import express, { Application } from 'express'
 import cors from 'cors'
-import userRouter from './router/user'
-import jwt, { JwtPayload } from 'jsonwebtoken';
+import userRouter from './services/user'
+import jwt, { JwtPayload } from 'jsonwebtoken'
 import * as dotenv from 'dotenv'
+import db from './models'
+import { users } from './seeders/users'
```
Comment (hpmax00) [inline src/main.ts]: We are supposed to have a script to create users data

File: src/models/index.ts [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+const Sequelize = require('sequelize')
+const process = require('process')
+const basename = path.basename(__filename)
+const env = process.env.NODE_ENV || 'development'
+const config = require(__dirname + '/../../config/config.js')[env]
+const db: any = {}
+
+let sequelize: any
+if (config.use_env_variable) {
+  sequelize = new Sequelize(process.env[config.use_env_variable], config)
```
Comment (hpmax00) [inline src/models/index.ts]: It's better to put the db  connection in a separate file. Cause there may have ssl config, connection pool or debug, and other configs

File: src/routers/loginController.ts [hunk 3]
```diff
@@ -0,0 +1,20 @@
+import db from '../models'
```
Comment (hpmax00) [inline src/routers/loginController.ts]: Controller or route is used to handle the rest api path. Then controller call the service to run the specified function. We are putting the function in wrong directory. They are supposed in service directory

File: src/services/user.ts [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+import { login } from '../routers/loginController'
+
+const router:Router = express.Router()
+const validator = createValidator()
+
+// login
+interface loginSchema extends ValidatedRequestSchema {
+    [ContainerTypes.Body]: LoginInfo
+}
+router.post('/login', validator.body(loginInfoSchema),
```
Comment (hpmax00) [inline src/services/user.ts]: And here, they are supposed in router/controller directory

**Comment by hpmax00** [comment]
Looks go to me

**Review by hpmax00 — APPROVED** [review]

**Pull request merged**